### ISSUE TITLE
HTMLPurifier cache folder 777

### ISSUE THREAD

**Issue by sergiu-popa** [symfony/symfony-demo#243]
Just installed the demo application using the symfony installer, set up the permissions for logs/cache folder got this error:

```
An exception has been thrown during the rendering of a template ("User Warning: Directory /home/user/Sites/symfony/vendor/ezyang/htmlpurifier/library/HTMLPurifier/DefinitionCache/Serializer not writable, please chmod to 777") in blog/index.html.twig at line 14.
500 Internal Server Error - Twig_Error_Runtime
1 linked Exception:

    ContextErrorException » 
```

**Comment by javiereguiluz** [comment]
This @sergiu-popa thanks for reporting this error. I haven't seen this error before and nobody has reported it before, so I'll try to investigate if we can do anything from the Symfony Demo app or if this is something out of our control because it depends on the operating system permissions.

**Comment by sergiu-popa** [comment]
I'm using LUbuntu (`lsb_release -irc` returns **Ubuntu 14.04** trusty)

**Comment by lfjeff** [comment]
I think this is the same as issue #281.

**Comment by javiereguiluz** [comment]
Closing it as fixed by #28. It will be available when the next stable version is released.
